src/router.ts

```diff
@@ -47,17 +47,21 @@ class Router {
     const date = new Date(dipDeposit.expirationMs);
 
     // TODO: Update this for other types of assets
```

Comment:
remove todo now that we only hardcode
USDC, which we said we will be safe to always assume
E, since we arent planning on changing from european
P, since we arent planning on doing cash settled